### ISSUE TITLE
Switch feature request: pass through props

### ISSUE THREAD

**Issue by gavrichards** [tailwindlabs/headlessui#84]
Hey team, great job with the new components.

I'd like to use Switch in our app but I'd like to be able to pass custom props like 'aria-describedby' so I can put a description under the switch and have screen readers read it when the switch is selected.

Could you possibly make it so any extra props are collected and spread on to the underlying button?

Thanks

**Comment by RobinMalfait** [comment]
Hey! Thank you for your feature suggestion!
Much appreciated! 🙏

Are you talking about React or Vue here or both?
But also this should already work, what issues did you experience?

Example: 

- React: https://codesandbox.io/s/headlessuireact-switch-example-forked-5cgmz?file=/src/App.js:538-609
- Vue: https://codesandbox.io/s/headlessuivue-switch-example-forked-6uet5?file=/src/App.vue:596-665

**Comment by gavrichards** [comment]
React. I'm confused by your example - it looks like you've put the description in the aria-describedby attribute itself, when it should be the id of another element.

**Comment by RobinMalfait** [comment]
Uh yeah that was bad on my part. But I just wanted to showcase that we already passthrough all the props. So I will close this issue because it is not relevant anymore.
